client/state/reader/start/selectors.js

```diff
@@ -19,6 +36,16 @@ export function getRecommendations( state ) {
 }
 
 /**
+ * Returns recommendation IDs.
+ *
+ * @param  {Object}  state  Global state tree
+ * @return {Array} Recommendations IDs
+ */
+export function getRecommendationIds( state ) {
+	return map( state.reader.start.items, 'ID' );
```

Comment:
right now this returns a new instance on each call. it should be memoized against `state.reader.start.items`